### ISSUE TITLE
Low success rate of custom robotic arms in PickCube-V1 tasks

### ISSUE THREAD

**Issue by gaojianpan** [haosulab/ManiSkill#1228]
https://github.com/user-attachments/assets/f53320b7-c7ad-4453-9ccc-c1589fb71f3f

Thank you for your great work, I recently encountered some questions while customizing the robotic arm, and I hope you can answer them.

 **The problem** is that the task success rate of the custom robotic arm after being trained by the ppo.py  in the PickCube-v1 task is very low, and it does not increase with the increase of the number of training steps. 

**The corresponding phenomenon** is that the robotic arm in the test video cannot remain static after pick the cube and continues to rotate at the 6th joint (as shown in the video below).

I tried to fix this myself, tried some solutions, but none of them worked well. 
1. Optimize the self-collision problem of the robotic arm and eliminate the collision of the front and rear links of Joint6, write an SRDF file imitating the widowxai robotic arm; 
2. Limit the [lower, upper] range of the corresponding joint6 in the arm_pd_joint_delta_pos; 
3. Modify the material so that it corresponds to trossen_black; 
4. The robotic arm is initialized to a good pre-grabbing position and the block is placed within the range where it is easy to grasp;
5. Modified the same random seed and training parameters as the widowxai robotic arm;

The most relevant posts to refer is this(https://github.com/haosulab/ManiSkill/pull/904)

I think the purpose of solving this problem is that I think the custom robotic arm is built correctly when it achieves the same good effect as other robotic arms on the ppo.py of baselines.

ps: The version I adopted is manskill 3.0.0b21

[assets.zip](https://github.com/user-attachments/files/21828571/assets.zip)

[agilex.py](https://github.com/user-attachments/files/21828583/agilex.py)

[pick_cube.py](https://github.com/user-attachments/files/21828585/pick_cube.py)

[pick_cube_cfgs.py](https://github.com/user-attachments/files/21828586/pick_cube_cfgs.py)

**Comment by StoneT2000** [comment]
The reward function might need some tuning or the RL hyperparameters need to be tuned for this. It seems the success rate is low because the arm doesn't stay still at the end. 

You can try
- Increase ppo training batch size
- Increase the weighting of the static reward component (and also increase the reward for success as well)
- Increase the number of steps per env taken in a rollout (the default baseline uses something like 4 or 8, which is very small. These were tuned for the Panda arm)
